e2e/client/playwright/custom-filed.spec.ts

```diff
@@ -0,0 +1,205 @@
+import {test, expect} from '@playwright/test';
+import {Monitoring} from './page-object-models/monitoring';
+import {Settings} from './page-object-models/settings';
+import {restoreDatabaseSnapshot, s} from './utils';
+
+test('creating custom text field', async ({page}) => {
+    await restoreDatabaseSnapshot();
+    await page.goto('/#/settings/vocabularies');
+
+    await page.locator(s('metadata-nav')).getByRole('button', {name: 'Custom text fields'}).click();
+    await page.locator(s('metadata-content')).getByRole('button', {name: 'Add new'}).click();
+
+    await page.locator(s('vocabulary-modal')).getByLabel('id').fill('textfield');
+    await page.locator(s('vocabulary-modal')).getByLabel('name').fill('custom text field 2');
+    await page.locator(s('vocabulary-modal')).getByRole('button', {name: 'Save'}).click();
+
+    await expect(page.locator(s('metadata-content', 'vocabulary-item=custom text field 2'))).toBeVisible();
```

Comment:
expand these tests to also add the new field to content profile and ensure that new field is visible in authoring.